documents/models.py

```diff
@@ -11,7 +12,18 @@ class Document(models.Model):
     commit_date = models.DateTimeField()
 
     def __str__(self):
-        return '{}{}'.format(self.path, self.filename)
+        return self.path + self.filename
+
+    @property
+    def github_view_link(self):
+        return 'https://github.com/{0}/blob/{1}{2}'.format(self.repo.full_name, self.repo.wiki_branch, str(self))
```

Comment:
The named method would just call `str(self)`. Is an extra level of needless abstraction really important to you?